### ISSUE TITLE
Add support for fetching sources via ipfs-js

### ISSUE THREAD

**Issue by edisinovcic** [ethereum/sourcify#318]
Add option to find latest ipfs hash and then fetch sources via ipfs-js for a specific address.

**Comment by edisinovcic** [comment]
Something like this or simmilar can be used: https://www.npmjs.com/package/js-ipfs-fetch

**Comment by chriseth** [comment]
I think it would be very useful to have a function that returns the natspec when provided with the tx payload and the address. The sources should only be a link instead of actual data, shouldn't it?

**Comment by edisinovcic** [comment]
> I think it would be very useful to have a function that returns the natspec when provided with the tx payload and the address. The sources should only be a link instead of actual data, shouldn't it?

Yes we can do it this way definitely, makes more sense for this use case.
